packages/react-server/src/ReactFlightServer.js

```diff
@@ -379,8 +379,8 @@ export type Request = {
   taintCleanupQueue: Array<string | bigint>,
   onError: (error: mixed) => ?string,
   onPostpone: (reason: string) => void,
-  onAllReady: () => void,
-  onFatalError: mixed => void,
+  onAllReady: void | (() => void),
+  onFatalError: void | (mixed => void),
```

Comment:
This doesn't follow the pattern of the others and should definitely not use `void` as the empty value since that can cause different hidden classes. We always use `null`.

But I think in this case we should probably just stick to the pattern and assign a noop so we don't have to check later.

Reply:
Notably there's opportunity when the passed in options get compiled that it can just compile it out to a known value. Either if it's always passed or if it is never passed.

Reply:
Which means that we probably need to add another field then.